app/static/settings_style.css

```diff
@@ -0,0 +1,147 @@
+/* Settings page */
+
+.sub-title {
+    font-size: 1rem;
+    padding-top: 0.2rem;
+}
+
+.settings {
+  color: #222831;
+  display: flex;
+  font-size: 0.9rem;
+}
+
+#settings-left {
+  padding: 1.5rem 1.5rem;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(230, 230, 230)
+}
+
+.settings-headline {
+  display: flex;
+  width: 200px;
+  padding-bottom: 1rem;
+}
+
+.settings-layout {
+  display: flex;
+}
+
+.left-options-bar {
+  display: flex;
+}
+
+.settings-main article {
+  margin-bottom: 2rem;
+}
+
+.settings-options {
+  flex: 1
+}
+
+.settings-options ul {
+  list-style-type: none;
+  padding-left: 0;
+  padding-right: 1rem;
+}
+
+.settings-options ul li {
+  margin-bottom: 15px;
+  font-weight: bold;
+}
+
+.settings-options ul li a {
+  color: #222831;
+  transition: all .3s ease-in-out;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.settings-options ul li a:hover {
+  color: #5786f5;
```

Comment:
You can combine with the active def below (`.settings-options ul li a.active`)

Reply:
yup